core/app/[locale]/(default)/product/[slug]/_components/wishlist-button/add-to-new-wishlist-modal.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+
+import { useSearchParams } from 'next/navigation';
+
+import { NewWishlistModal } from '~/app/[locale]/(default)/account/wishlists/_components/modals/new';
```

Comment:
Should this file be moved to `core/components` since it is reused.

Reply:
@jorgemoya I was considering it as well, but this is likely going to be the only place that it gets reused 🤔

Reply:
We could probably move all of the wishlist modal definitions to `core/components`, as well as the share button? 💭

Reply:
Looks good to me!